models/adapter/marketo__lead_adapter.sql

```diff
@@ -1,3 +1,3 @@
 {{ config(materialized='view')}}
```

Comment:
What is the purpose of this model?

Reply:
I wasn't able to find much, but the commit comments say [add tests for intermediate](https://github.com/fivetran/dbt_marketo/commit/4377a5eac2a54bd1eebe6aeebe4fe82055b5df7d). I think it might have been necessary with the old setup, but after adding int_marketo__lead, I was able to remove the adapter model and convert the int model to a view.